Tools/ssl/multissltests.py

```diff
@@ -535,17 +573,38 @@ def main():
         # check for configure and run make
         configure_make()
 
-    # download and register builder
+    logging.basicConfig(
+        level=logging.DEBUG if args.debug else logging.INFO,
+        format="*** %(levelname)s %(message)s"
+    )
+
+    versions = []
```

Comment:
You could have a dict of classes to a list of versions instead of storing a tuple of (class, version)